.travis/linux..install.sh

```diff
@@ -15,3 +15,9 @@ else
 fi
 
 sudo apt-get install -y $PACKAGES
+
+# Fix header
```

Comment:
> Non-Travis builds may not have sudo permission.

Can't we leverage a cmake task which places the custom header in user-space?  Even if it's fixed upstream it will take a while for it to make it to the various -dev channels so it seems like we could hit two birds with one stone and it's not the only header we write at configure/compile time.